lib/Page.js

```diff
@@ -80,6 +81,18 @@ Page.prototype.prepareTemplateData = function () {
   };
 };
 
+/**
+ * Records all h1-h3 headings into this.headings
```

Comment:
@nusjzx Note the change in the first line, from https://github.com/MarkBind/markbind/pull/240#discussion_r192317692:

> ```js
> /**
>  * Records h1,h2,h3 headings into this.headings
> ```

Reply:
noted